modules/custom/az_alphabetical_listing/az_alphabetical_listing.module

```diff
@@ -27,7 +90,7 @@ function az_alphabetical_listing_theme_registry_alter(&$theme_registry) {
     // If the template has not already been overridden by a theme.
     if (!isset($theme_registry[$key]['theme path']) || strpos($theme_registry[$key]['theme path'], 'themes/') === FALSE) {
       // Copy default node registry entry for node entries.
-      if (strpos($key, 'node') === 0) {
+      if (strpos($key, 'node') === 0 && ($key !== 'views_view_summary__az_alphabetical_glossary_attachment')) {
```

Comment:
I think this check should happen for the `if (strpos($key, 'views') === 0) {` line, rather than the node-based if statement